components/phttp/ammo/simple/uri/decoder.go

```diff
@@ -37,15 +37,15 @@ func newDecoder(ctx context.Context, sink chan<- *simple.Ammo, pool *sync.Pool)
 
 func (d *decoder) Decode(line []byte) error {
 	if len(line) == 0 {
-		return stackerr.Newf("empty line")
+		return errors.New("empty line")
```

Comment:
We ignore empty lines in provider's [main cycle](https://github.com/skipor/pandora/blob/ee18af8390fd1e59d4bf98e7c1f5d3a4b0b02a52/components/phttp/ammo/simple/uri/provider.go#L52) (and this is OK). This code must never be executed. Maybe panic here?

Reply:
The error will cancel shooting anyway but will contain more context. Line number for example.